scripts/host-check

```diff
@@ -314,6 +320,23 @@ def _get_cgroup_version() -> int:
         raise Exception("Unable to detect cgroup version in use")
 
 
+def is_al2023() -> bool:
```

Comment:
Sorry to comment on this again - wondering whether to make this more generic and just have it be "is_aws" and just check for amazon (rather than a specific AL version)

Reply:
The TuneD version used in our packer has been up-revved for AL2023. As a result, an additional cmdline arg `rcupdate.rcu_normal_after_boot` has been added due to a change in the `realtime-virtual-guest` profile (I copied the latest version of the profile for our grubby args).  To handle this, I could:

* Vary the cmdline arg check depending on if AL2 or AL2023
* Drop this requirement unconditionally
* Not support AL2

I have gone with dropping this arg as a requirement.